pkg_api/utils.py

```diff
@@ -306,3 +306,102 @@ def get_query_for_conditional_get_statements(triple: Triple) -> SPARQLQuery:
 
     # Cleaning up the query
     return _clean_sparql_representation(query)
+
+
+def get_query_for_remove_preference(pkg_data: PKGData) -> SPARQLQuery:
+    """Gets SPARQL query to remove a preference.
+
+    Args:
+        pkg_data: PKG data associated to a statement.
+
+    Returns:
+        SPARQL query.
+    """
+    blank_node_id = "?statement"
```

Comment:
Isn't it always the same? Can we make it a global variable?

Reply:
I created a global variable `_SPARQL_STATEMENT_VARIABLE` for `?statement`, please check that it does not impact the readability of the queries.